invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/components/CommunitySelectionModal/CommunitySelectionSearch.js

```diff
@@ -43,18 +43,23 @@ export class CommunitySelectionSearch extends Component {
         toggleText,
       },
     } = this.state;
+
     const {
       apiConfigs: { allCommunities, myCommunities },
       record,
       isInitialSubmission,
+      CommunityListItem,
```

Comment:
minor: instead of passing the component, is there a way to achieve the same effect with overridable? We don't usually pass components around as props (cc @kpsherva in case there's any issue with using overridable here).